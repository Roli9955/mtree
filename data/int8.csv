id;val
0;7027890636298843910
1;-3089417368928478640
2;-1887233656988596533
3;8503237526526644194
4;1737164665112153261
5;-6857391772919546239
6;2416257134116255710
7;-2078459808602527200
8;-1480394550698824013
9;-2814126248584352186
10;3210866242841355338
11;-3901183210596684107
12;-4901181317369918620
13;-7677576934327127331
14;1267052794694569914
15;-6819001237595868153
16;7320854785678451936
17;5112035791124693326
18;-6050490617295258498
19;7921393305515067706
20;-1963880674005274187
21;-9007854174816107037
22;4948860233555961890
23;-8421394666751241622
24;-2022021217654970818
25;-1616371871971266151
26;7313035512260873163
27;-8822153613971582582
28;8944555800515542696
29;-3288320185995029425
30;7742907252961775763
31;6169210826979670565
32;4149016220346119601
33;-1104274127297537201
34;7377268009009183261
35;-4757821016456048590
36;-1691427585431011707
37;-5924443441768071704
38;-8258305103234058759
39;-8823091576532588077
40;7722782284743396903
41;-1856184039786766740
42;-8370310050540142252
43;3071549986367977674
44;-751447336288176123
45;2764119086148975027
46;7697013690225013256
47;-6499476398689241662
48;6409307763826903603
49;-6634597890492122877
50;-3950638047399311279
51;6147822468343371533
52;1937164502733829392
53;8682018065601616798
54;-7065746631394023202
55;6441867516384009395
56;63129973535589275
57;-8462584284285263775
58;-2046685384903183890
59;3724731324960450353
60;-7070849770759288959
61;5490962698454245653
62;-7595438593199488531
63;3317839499441000780
64;7499404633129520148
65;-918352875723314425
66;-6513467205527885984
67;-6527729659360525241
68;-8249438412393511334
69;-3320191511716979914
70;-5071233780783326129
71;149441807218355765
72;1938739494169696427
73;7631932078155385562
74;8779781125621423870
75;-3370612214642109301
76;-6726271117530006232
77;-8541111956395720059
78;3028496901893253022
79;-6279812045459381145
80;-6457054765370737450
81;-619624030883263950
82;6797039909036716967
83;-1564786906259988219
84;5624172190249209410
85;5678037457202534182
86;8660379387957931475
87;-6033793919926287489
88;659416081028278750
89;819090504133529495
90;1601380150865518567
91;2109527875162646511
92;914626825059462831
93;-8903559173518512410
94;-514740180852629614
95;-8001371764460375424
96;718827806998665098
97;4531740984772185970
98;-3039669604225584088
99;-8561579534807838126
100;-6181468827578799220
101;-5783133990441240584
102;-64662715002780145
103;-7499138065876465623
104;-5660874215577262148
105;7405954891433234930
106;4831505583939985180
107;-1644812074302796753
108;2658599299151375853
109;-8707995431727451370
110;6047186028250752630
111;-4378383734352435148
112;4927181049050941760
113;-8040539517209950809
114;-8446536929710674828
115;-5788835766518478169
116;4156916637789224871
117;-1273469882393272408
118;-8665903012310815064
119;-4270388421136050488
120;1042533454733310829
121;4001261109138893116
122;5541492417722891644
123;3506027483452600383
124;3324040098935637780
125;1410199304928466755
126;2527783811003374339
127;-8206506950003131330
128;-834500426202424975
129;9041659202953612357
130;-7230870029354420124
131;-5505911059135917908
132;4204067082654572272
133;3528478656077856926
134;5895934681413881519
135;1486694962011808143
136;8744692265639392881
137;6458881297693669471
138;8682649738698966291
139;-8237240602557346002
140;-3506358640209181638
141;7837025280840987985
142;4207057851462847019
143;6202427109266936847
144;3687660984098962086
145;-6668166956747000832
146;8218800668309537670
147;-6375162491422951697
148;2606031994833166902
149;-1993008353393257378
150;2952922358487048433
151;8352588643868483043
152;522370511688626002
153;-7413309056646262425
154;2483008796505106656
155;-5788728717449250041
156;951639071386295338
157;-6924457305401312465
158;762378911181279426
159;-1419795305761821726
160;457738935269413473
161;-6738740289501476918
162;-8378375273953278071
163;7620986254621876332
164;1465840886947243041
165;-7144087258448756144
166;3837359338671715677
167;-685237078002964684
168;5650990032006039654
169;-406531042888468504
170;-4576270414727805731
171;-1450535738878537163
172;-8719616982994602497
173;-1749903226080903477
174;-8297889359889383041
175;-3566587935020062473
176;4764590794977379351
177;-5955379972679741445
178;8978728106171847086
179;7086413257988750398
180;132630883743800088
181;3786021640239986844
182;-1120468985660530553
183;-5318553867523446721
184;-8063677970483586441
185;2638743535735272878
186;5205514949707080033
187;7426334427172911336
188;-3005098137154681671
189;3542688459858493934
190;-3788104168429658332
191;5097536939616114161
192;8072572981428931528
193;2324982677092976935
194;3180268536334425374
195;-9031411508501989314
196;-9150451661631610848
197;7758088519250923424
198;-6672562489606809671
199;9221751993739881557
200;-332792476191595501
201;-7588314361660036938
202;3280741648092393773
203;-6591927460322445817
204;-758960110358356911
205;7207571060015437723
206;6598897151659912026
207;3552818280469259873
208;-5554488273795838239
209;-6677951517896380715
210;-4159238160171753924
211;-9051351473321067164
212;8361079798799818265
213;5823064976249850380
214;-35110513303017830
215;-8100832614540235063
216;-802719002021504521
217;-3374217343468893914
218;-7622708439086745237
219;4831831901367434298
220;209621319802671024
221;-1863977629777969039
222;-2924591866211751021
223;-728749032372585228
224;-4631653098641381626
225;2265382916110189407
226;-1688591989815045799
227;-1401800772102632138
228;-8384045896554832058
229;-5837358585049762122
230;6378658967878250228
231;2813047043145712365
232;3911913871537142055
233;-301619168759329321
234;1295169491318125205
235;7528153109859370847
236;117710244333596354
237;1711974007460390960
238;6827081976211392516
239;7773360662890419668
240;5990823026455517778
241;2872459969458984806
242;-8473644637575531717
243;8528056670666115411
244;-359774450588863637
245;4132122589494510307
246;1683294725812728264
247;4863623529545024612
248;3892196144297661011
249;-8440909298518707722
250;-5915007177653801409
251;9105085609526742368
252;-3049871521562444978
253;-9097466773132289190
254;1117400700712222206
255;8027589811189351716
256;-6084482906219293211
257;-2687105765095399806
258;-5067482377764359395
259;-2250383937326722374
260;851029649015125623
261;658657077996144808
262;487100843684471150
263;4247868534811473158
264;-3679937721730534740
265;-2470653314147608626
266;3175197414594492268
267;-654356417083109360
268;6050545112410036934
269;-5340314025651501076
270;9190070455355747361
271;1835570162922473731
272;8308798506163487284
273;-8112391864301493582
274;-8619592938540869079
275;-696627091407940576
276;-4765528357698350993
277;8910670997884991033
278;-6136982170270214151
279;-873603750423941626
280;-5097734089264053654
281;-35380184726420105
282;-3691418928821311286
283;-8843339739623129831
284;8374705854472133686
285;-2303112516766390495
286;73042160445465294
287;1520442082070416033
288;4174618851520859422
289;-97948949198204318
290;6953032607462012411
291;-1292610608015936009
292;9147304336333926314
293;8742276873512059385
294;-7880515969617779700
295;258839032839771736
296;-4643987367619401643
297;-3876065076683306559
298;-6866774135040049191
299;-5945131812412226351
300;5554804887956164018
301;8685588755040631545
302;7797825719335048723
303;-7375656408599499830
304;-3814779143680138333
305;7887016448149221446
306;-179930862136344490
307;461945604722881320
308;-9081418235627469246
309;-1121362950260322369
310;7695091499443987029
311;-8099494657153599777
312;-4624588497727397678
313;-5136463797682879632
314;5493002626062713443
315;3526135492933314552
316;8880382001778005014
317;-1613733123309422644
318;7414121277898965263
319;-3006345556391391388
320;3755584716901754766
321;6878241559179842609
322;-5424703650719263989
323;-8564856839669964347
324;-4560748720023810526
325;-3549608376204630534
326;6224518188330994443
327;-5432362138046423697
328;-2217481734220843936
329;-5007762852269800088
330;7506264960992016051
331;8541405780272728113
332;-4085305350992001700
333;-1479950793050962515
334;7729481741174168291
335;6072752049280351358
336;7409186188319008805
337;3443112138860932338
338;-4379231696091981158
339;1778914206626380657
340;757711615791130036
341;-2451126276665550647
342;9025792593558541183
343;-5723577106695534457
344;622612927525618199
345;-1439181629825173429
346;-2785488529584197759
347;7226875180380712352
348;1862975113284815898
349;-8174979891445576524
350;-2695537829155507123
351;-2133786637425698657
352;3409702221945288579
353;8540243680651635821
354;5945872485438634511
355;5201507426286138930
356;8605170196986764254
357;6773502502807979807
358;7869503458539840630
359;-1389485528891365695
360;7187039238222661909
361;-1494190671490791441
362;-2638273424166724935
363;-725488648709157566
364;-1390738622804474687
365;-7959982431740714612
366;-7466993439413975698
367;1397290993806541023
368;-6127098997457226740
369;8189747015443466308
370;-2758217332938266192
371;-7814482514431133919
372;-2084780857894382367
373;2952667185622911762
374;-6313547752571037688
375;3882054774993363465
376;-3347848820257889153
377;-1336509197637036197
378;6191105425914965508
379;8296420823986072431
380;-6455926639543258002
381;-1236873899290460199
382;-1698877575316830605
383;-7176489987775796951
384;-2094088872543786254
385;2981034634529012262
386;2023052264861195889
387;5688853125396421099
388;6424850905726848118
389;3590560060163204774
390;8659330664056396014
391;-1305413754438675588
392;4088717230786670698
393;-8621696601199062141
394;-9199225613817980845
395;6589334536292815670
396;-7470957332528996685
397;-2600260077547151016
398;5200125286083938809
399;-2659234294926035747
400;-4097229243898392620
401;8511845867026696244
402;-6378282448784027722
403;-234441962800181862
404;3392904895283421722
405;3079138436305753817
406;-2693141736743830034
407;4590874488149390660
408;-7853778540767826862
409;-4041467044632501699
410;3860993580350165939
411;1304684708600785773
412;-5285666019294467213
413;1864448678270263347
414;3653808133725990557
415;-4233674069426220395
416;7418659488119724414
417;-1660346809347760255
418;-3815989017814110677
419;8869913862361295750
420;1414066380667407827
421;-7290741745483489039
422;-2653509877480246669
423;-578779567699703063
424;947076257590528189
425;-1128405920635827518
426;2837170017438854373
427;-6091621036682774298
428;8074579850177992094
429;8488448602858054723
430;-6346224861070404492
431;-2733183791703941711
432;-6043250204947394364
433;2608489863076595630
434;2331161015307916992
435;-4416910480003496462
436;7901809797175985349
437;2064083950493961777
438;-5223393354000049332
439;7786233268652326328
440;8939691932148798031
441;7605738328440136800
442;6121917405800974066
443;5974552465983354670
444;-4406217711503294678
445;-6370686808918073341
446;-4720088193531646277
447;-8842402550076712754
448;7162618988715048744
449;-1910515136887936060
450;8016511016788193258
451;-3117344420778430020
452;6549883716077430313
453;-8467188277732912838
454;455371223641945445
455;4380343987629188992
456;-6891757257852820972
457;-8029269917000560633
458;-672866838304456154
459;-7281297677958525583
460;5853753325263767778
461;-5230723111691639439
462;-7744259451901905636
463;1154403761041730791
464;-1446929530496002151
465;2218026286721494454
466;3694033780133733195
467;2490381079206504206
468;748196203711375205
469;670561060838831412
470;-6812098105207554165
471;-5564761167078415846
472;-6555333795838132137
473;-659108868185305899
474;8945092431911385220
475;-2597077716886554185
476;5027309712845671103
477;-379009316807283925
478;-820389313574825785
479;8584654221668306027
480;-5825107104286078167
481;3444266644709652711
482;-2461718891930883692
483;8681381186706195710
484;6097162341090212278
485;7912145924914323697
486;-8616654699272071592
487;74246901143428231
488;-4012878526485362326
489;7013388888674827371
490;-333827040695001611
491;-5575582004654446302
492;-7809790499889816413
493;1326512278336800145
494;-6250952697657999907
495;-1446812869058282749
496;4592240644055143307
497;-7111304364842276952
498;6570944210422234550
499;-8192279383157026006
500;6025660878754764661
501;662769764997120570
502;8882684379052604156
503;-956233740987922754
504;636164573634754294
505;4370094812075773051
506;7987986429096323554
507;-4479962620844622712
508;-8649779565552444630
509;-423296506735770699
510;-634151590564196249
511;-6429000505099903431
512;7508813598188537235
513;-4451436351536614363
514;-4077921320030648191
515;2352439189717304605
516;7411556924283020115
517;8332615519367914156
518;2174571642512257326
519;5249962854640979620
520;-365753055307556878
521;104585205057166680
522;-8275589766601403166
523;-3073147708535686314
524;1618483523519503704
525;-2747648695271587106
526;-5220508837946548631
527;6492087140126170842
528;-703993645985717089
529;-436942863281157262
530;-6048667062938364183
531;1629015660775137037
532;-1456143334469984495
533;3777997996335213478
534;-6197769567766972769
535;-550646308007288053
536;-1954846668056766494
537;-8941481180400217317
538;-6666487091910986731
539;-8258383657045499141
540;-6611361485086767092
541;6504034569879161908
542;1348146369575481815
543;9023377390812859726
544;7329992490180715296
545;-5583257409889669136
546;-3881081402715975791
547;-7404515722591275231
548;333791406257482460
549;-3908597165980378863
550;-1875315628425106330
551;-2877575413291365937
552;2140839266398577898
553;541036604946913257
554;7120931363537761627
555;4475430286551012471
556;-6438084518680396959
557;-6314451024057411807
558;2178943290719786770
559;695437952490333328
560;-1139731177641364953
561;-5465566618625183331
562;3120589545346948484
563;4065513780423450956
564;109160495612256116
565;5395833705958816355
566;4754965351271062339
567;2415188601009484591
568;-1629561559345395097
569;-8856266799949356801
570;-3046015420433678178
571;-9149107526341662467
572;5435785891645920861
573;-3350073636550975955
574;-356760765338868949
575;-1864257538083859864
576;-6836460856298765761
577;4520411121728447150
578;1083030856743141450
579;9138935517695486642
580;8179863628974250834
581;-4628042775791431948
582;-7781928723251846913
583;-3185048476997541457
584;484040741854995742
585;-7714095430693841197
586;3937858824709598934
587;2931899379133036720
588;-477318123153356057
589;8796494726406204709
590;4943876008808568673
591;4604362190033418323
592;7873359532101620310
593;-1368915840842176098
594;-303584399279812882
595;-6571881145988194426
596;2587712497632119998
597;3726648515626927751
598;8395594840214221784
599;-2485471902337585958
600;-6834135071801323297
601;-5356993838933243407
602;4346725047897341911
603;1643865458314698374
604;4716369961946469619
605;-1776156969092970277
606;-739908298790871851
607;-7906130635374326654
608;-9025823897830487073
609;9160113040967200224
610;-5922039882937004953
611;-5414424966433167228
612;4345892001096872534
613;7808176478761670469
614;7782121079928768276
615;-7903904106237898856
616;-8660780741789109510
617;9137146617663360458
618;-7379748888318491403
619;2613253142009787679
620;-144938763582709414
621;-2386248722034592154
622;-5591119134745720049
623;2898343391335721380
624;-6770004256273768851
625;9057654952170464968
626;-8080959724207128878
627;-8431305098839490755
628;7584208388345239321
629;4850741078517459690
630;1437201653149100417
631;7450982754340255907
632;7080011017869034069
633;-5146874542228218891
634;3970824454669608484
635;-7063520321643652073
636;-1269041874107697191
637;462155150641472159
638;3645276256319579018
639;-8523293554739071575
640;7029312553494291944
641;7175303864399441565
642;-7247576365204605397
643;928632202850164923
644;-7206406563444771270
645;-8323474864635106300
646;2349162195722234972
647;-4344226145259122411
648;3488980718532210737
649;5186840178942387636
650;-4676842393751463280
651;-5273875211612310347
652;5312215473460714390
653;-2317829974994618505
654;-1915860493066580421
655;-3121414067576538043
656;-1126755940866728852
657;-1330277388741856922
658;-3735389117498394778
659;-5836869284262172593
660;-7651773268626848356
661;-13167393176535494
662;1678748019901707335
663;8338573159726342934
664;1216018101750354256
665;8499091425790506910
666;5087055762395620890
667;-8844435017494690679
668;-7222731306146438604
669;5643761636679094019
670;-2245102793843174192
671;8549261941632983168
672;1662543595550130604
673;-4878033285045797975
674;4056557333022458406
675;-6812973571935400345
676;4798454572237046480
677;4154052301772002259
678;-2499921392921487882
679;-3206260887894398334
680;-3194393086181839439
681;7927982216155853540
682;8758828631358969443
683;-4993574533148489966
684;-3832007306199976890
685;1162585385248342307
686;-1828770548763716857
687;3282021377656381606
688;-6135497724802941614
689;-7520686369024908962
690;2561713839839559384
691;-6709660028917272565
692;-929997920959565280
693;-6022153514522348360
694;-6533154934346936137
695;-3459091856386940497
696;-3161989040508439475
697;9144549604377365513
698;-5199520563286093497
699;-7897416364212945516
700;-206753790225332764
701;2294110975519115173
702;5133027547634139201
703;5649479648449416667
704;-1833933640893941614
705;-8206444681579403111
706;5793861641898971456
707;-8674566184497843227
708;-2345721649542974177
709;-7757151683135122302
710;-4234155150837901044
711;3806442974963684677
712;-1913021736410199178
713;-8058819918507593441
714;-1876606506509017353
715;-6069311845928698678
716;1412986210590718074
717;-1551376202916638762
718;-8380737424977549780
719;-1823243573227092116
720;-2780142850843006749
721;6382479894659479706
722;7135754260213583682
723;6001334403157877467
724;752001410200395750
725;-7595279396597690436
726;7160264211052226512
727;-8436922700256107012
728;1483251085831649105
729;-8921241468617158454
730;7408053921795455
731;8185127566856296587
732;-7808603620728139353
733;-7544539921517690727
734;-2116972206046546647
735;7972472278690080017
736;-1913710706480760145
737;-4600832159652553237
738;-5278901659298303733
739;3422260871647044791
740;3198505301872208959
741;569854452784107807
742;-4992643864115207232
743;3364825466530597223
744;2911485637576625135
745;5871142138339380603
746;-245533357844475622
747;-545187440133670611
748;-1764632874590891875
749;8651795779336949498
750;2032486422367522208
751;-363177285269130203
752;-1253700936332717936
753;2614530908794370263
754;2495402152744406722
755;-2634867460056595003
756;-3416956913257986715
757;1139279484555723327
758;-7686712767254708834
759;-4369234725381644081
760;-8019548030101590373
761;-43095142412239905
762;4379990258493447899
763;3592262957750301048
764;-5931031988125378264
765;-869905866634908461
766;3382911424601291989
767;-1718232114257953987
768;-2799515930903967125
769;6197875329174156538
770;2683555691089801674
771;9032998010932329437
772;-1404002325962121602
773;-1448956216725793722
774;-3976978454772907425
775;1454893052298432165
776;-1119891900783558587
777;4094173364989356594
778;92397739678221903
779;-1257064923309894166
780;-8991975778210647261
781;8319503306968091020
782;-1196316340303598616
783;8491122016892633783
784;-2450062139615669581
785;6411636275732870462
786;-6119108699739230044
787;4449787528239613840
788;-2100153692820587584
789;4665727914854439350
790;8768605745597810692
791;1252404317511999503
792;5933168584515190638
793;-6348745581519481128
794;3171015488958169870
795;3280991485007520435
796;-1017733690598776513
797;-2536912970232438202
798;8845196654441681169
799;-1782760132911677317
800;-7150699312021694671
801;1215733738160677955
802;-9001972167867154770
803;7661614150555010829
804;-4593340054161354567
805;1314665044497306362
806;2554933836581859704
807;6745032686091038741
808;-820840846619818801
809;-357935358717639170
810;1983403687032029036
811;1578107342675596675
812;-9007636887665366171
813;1637478254138377614
814;-276912237127830044
815;-1017034190346383895
816;-9154372518954633988
817;-8809558079263858362
818;901658406810681175
819;4086111610000991286
820;9210448375583686857
821;-5629930309297361404
822;7548154370177765055
823;212242303098465120
824;4218706068818878597
825;-6748071459782526722
826;-8116906697158174290
827;6638830648508166695
828;-6797730034739872670
829;3309903098766642904
830;-7729894880685704085
831;-7376525169188652603
832;-3520588675067182810
833;-281561295399084195
834;-7707558347668314450
835;589259875315101583
836;6361850304240681652
837;3788539322636940216
838;-7987418335810925260
839;-534698416988520622
840;-2257214221205697529
841;5610770977133165464
842;-5460856461242711899
843;5778165621417238119
844;8652418470939420930
845;-21836474177794172
846;-1110394960513647668
847;-6846717251045310708
848;-5278308894803959359
849;27114581772520137
850;8797406951054324301
851;2054343674704415450
852;8264395511924370801
853;5568199432338882296
854;-7380485247971627186
855;3553558954446101562
856;-6017771223058228315
857;560543141033340440
858;-8847872406525685329
859;-9160372919495810157
860;5763849155410581996
861;-7759789653778552115
862;670794305769337539
863;8968721402532004130
864;6296108662302913126
865;7944050638340131642
866;3529120915868932782
867;2080258325085091407
868;-8866119768333943962
869;7188986382170284133
870;4818564273591644315
871;1713342371788747936
872;104232989219813210
873;6802481126225246760
874;-2458880912800521566
875;-2074775546866425761
876;261481245226498698
877;5608261896414869935
878;5649261683153357208
879;4515563230008204225
880;4830484697334894484
881;7824870216500668878
882;3643256350238367039
883;-7852540891983552458
884;-4407447439282958818
885;-4836153652217005218
886;4393987322796330166
887;-334435714214730855
888;4828048477582745182
889;-3642256590288532911
890;-5181217760568255220
891;5270819604881214035
892;-2956358702440423923
893;-9130299269050691753
894;-7151083961911178998
895;4639453092350546349
896;4950156995371151084
897;-3051578985630999416
898;-5408824908939898370
899;-8385921127902352720
900;-8583628086588187766
901;-8200140806392914698
902;-3475909031562793612
903;6603323775358610156
904;3487504557379542805
905;5366723999035821939
906;-8926206736409513376
907;-4764520897436543345
908;6378804757590165259
909;420580701566449914
910;-9198150525374483061
911;-5419431044383401265
912;-1459985307578054724
913;-4153957255626146781
914;7376648330219641185
915;-3722599827585187711
916;6811285646087030135
917;-7650852205710478008
918;-2004430909034798360
919;-2165758514984443474
920;-6622257165849365988
921;-6205872529193587048
922;915742378190094579
923;-5893811147603781320
924;4833176146052512681
925;-7468373642272318801
926;2975848636956360205
927;3978131016917699955
928;-5471485554382886243
929;-8316999381765266801
930;-4323984052590345776
931;-6950054952522870447
932;1467611095391486205
933;7319271502663987887
934;-3247590237026981560
935;2251547800530737375
936;2187930170526526935
937;9023339817143009962
938;6994534323548812903
939;-4933684840248984661
940;-1567680351555277102
941;-1982816340213720871
942;-1527595665028680424
943;-4888378856500767200
944;-2259222909177441959
945;-4630180944964464724
946;-7979652901953959150
947;7820866813763632190
948;6515725761851701148
949;-8148094913920787571
950;7175862915854703621
951;-4553227600254451357
952;-3361818979848578097
953;-2704743884097942087
954;-5867742205852416673
955;-6843452868041517024
956;-8077758293467619809
957;3290951403384789083
958;-5269087479012178807
959;1720048811304270945
960;-6427246789527700423
961;6270816986851412932
962;2962581896537752671
963;4867169703082735036
964;1472015621888473911
965;-6508969932512829347
966;-8576529130932477598
967;-6140933906954303064
968;8767971196293279470
969;7326355804141802092
970;8554001669586786780
971;6506774159716698803
972;-395783390516779003
973;128005235186304023
974;7010474715577595065
975;75519830301377907
976;3117414719448914643
977;1277642762287842926
978;-8375518384434425315
979;-4879624541094022160
980;468276372654209495
981;608545799918072774
982;-1287342857159634648
983;1040161422352715382
984;-7774434572532293731
985;-3286065164321536897
986;-1990867038985817814
987;-6692637465573043371
988;5108397965333488270
989;7357568689567686779
990;-4369072137073791110
991;775812514401888107
992;-8012975704230247179
993;-537115490034919898
994;-1903500074121838897
995;-5832374946660924138
996;3191519296375242422
997;7352981112756933142
998;-3796736957595291775
999;3348439464701854822
