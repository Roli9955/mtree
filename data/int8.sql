/* contrib/mtree_gist/data/int8.sql */

DROP TABLE IF EXISTS INT8_TEST CASCADE;
CREATE TABLE INT8_TEST (
  id  INTEGER,
  val mtree_int8
);
COPY INT8_TEST(id, val) FROM '/home/postgres/test_files/int8.csv' DELIMITER ';' CSV HEADER;

DROP INDEX IF EXISTS INT8_TEST_IDX CASCADE;
CREATE INDEX INT8_TEST_IDX ON INT8_TEST USING GiST (val gist_mtree_int8_ops);

SET enable_seqscan TO OFF;

SELECT COUNT(*) FROM INT8_TEST;

SELECT * FROM INT8_TEST;

SELECT id, val, (val <-> '0') AS dst FROM INT8_TEST ORDER BY (val <-> '0');

SELECT * FROM INT8_TEST WHERE val #<# '0';
