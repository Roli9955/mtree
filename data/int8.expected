SET
DROP TABLE
CREATE TABLE
COPY 3
DROP INDEX
CREATE INDEX
SET
 id |         val          
----+----------------------
  0 | -9223372036854775808
  1 | 0
  2 | 9223372036854775807
(3 rows)

 id |         val          | dst 
----+----------------------+-----
  0 | -9223372036854775808 |   0
  1 | 0                    |   0
  2 | 9223372036854775807  |   1
(3 rows)

 id |         val          
----+----------------------
  0 | -9223372036854775808
  1 | 0
(2 rows)

